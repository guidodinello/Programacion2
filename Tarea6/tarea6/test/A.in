apilar 10
cima
encolar 5
frente
insertarEnAvl 15
raizAvl
singleton 20
cardinalidad
asociarEnMap 2 2.0
valorEnMap 2
insertarEnCP 8 8.0
prioritario
Fin
