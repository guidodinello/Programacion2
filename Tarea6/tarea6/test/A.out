1>Apilado.
2>10.
3>Encolado.
4>Binario vacío.
5>Insertado.
6>15
7>Creado el singleton.
8>1.
9>Establecida la asociación.
10>2.00
11>Se insertó el elemento.
12>8
13>Fin.
